var/spack/repos/builtin/packages/amdlibflame/package.py

```diff
@@ -69,21 +70,32 @@ def lapack_libs(self):
     def configure_args(self):
         """configure_args function"""
         args = super(Amdlibflame, self).configure_args()
-        args.append("--enable-external-lapack-interfaces")
 
-        """To enabled Fortran to C calling convention for
-        complex types when compiling with aocc flang"""
-        if "@3.0: %aocc" in self.spec:
-            args.append("--enable-f2c-dotc")
+        # 3.2.rc2 version onwards amd optimized flags are encapsulated under:
+        # enable-amd-flags for gcc compiler
+        # enable-amd-aocc-flags for aocc compiler
+        if "@3.2.rc2:" in self.spec:
+            if "%gcc" in self.spec:
+                args.append("--enable-amd-flags")
+            if "%aocc" in self.spec:
+                args.append("--enable-amd-aocc-flags")
 
-        if "@3.0.1: +ilp64" in self.spec:
-            args.append("--enable-ilp64")
+        if "@:3.1" in self.spec:
```

Comment:
Here and below there are a lot of overlapping conditions. I just wanted to double check that this was on purpose.

Reply:
Yes. 

For example, configure option for 'FORTRAN to C calling convention for  complex types when compiling with aocc flang' was:

- For version 3.0:3.1 it is  "--enable-f2c-dotc" 
- From version 3.1 onward it is "--enable-void-return-complex"
-